README.md

```diff
@@ -46,6 +46,14 @@
 - [SOMAS](https://doi.org/10.6084/m9.figshare.14552697):  Experimental and calculated solubilities for small molecules. Originally proposed for the design of redox-flow batteries.
 - [Therapeutic Data Commons](https://tdcommons.ai/overview/): ML tasks that cover small molecules and biologics, including antibodies, peptides, miRNAs, and gene editing therapies.
 - [ThermoML Archive](https://www.nist.gov/mml/acmd/trc/thermoml/thermoml-archive): experimental thermophysical and thermochemical property data (in ThermoML XML format)
+
+## Target identification data
+- [Open Targets](https://github.com/opentargets/opentargets.github.io): is large-scale resource that uses human genetics and genomics data for systematic drug target identification and prioritisation.
+
+- [Probes & Drugs Portal](https://www.probes-drugs.org/data_sources): is an interactive, open data resource for chemical biology containing focused libraries of bioactive compounds including commercially screening libraries.
```

Comment:
```suggestion
- [Probes & Drugs Portal](https://www.probes-drugs.org/data_sources): is an interactive, open data resource for chemical biology. Overview of libraries of bioactive compounds (e.g., ChEMBL, Guide to PHARMACOLOGY), including commercial screening libraries.
```

It seems they only link to other databases; I tried to clarify this (might be useful to also list the open resources they link to here)